lib/tiappxml.js

```diff
@@ -810,6 +823,25 @@ function toJS(obj, doc) {
 					}
 					break;
 
+				case 'webpack':
+					const webpack = obj.webpack = {};
+					xml.forEachElement(node, elem => {
+						switch (elem.tagName) {
+							case 'type': {
+								webpack[elem.tagName] = xml.getValue(elem);
+								break;
+							}
+							case 'transpile-dependencies': {
+								const transpileDependencies = webpack.transpileDependencies = [];
+								xml.forEachElement(elem, dep => {
```

Comment:
How are nested dependencies handled? I see an npm package name used, but what if the package appears multiple times (different versions) in the node_modules tree? Does it handle scoped dependencies (i.e. `@babel/core`)?

Reply:
The values listed here will be converted to a regex using [this](https://github.com/appcelerator/appcd-plugin-webpack/blob/16cc5568bc4def2efdd627667b64620290224af5/src/utils/webpack.js#L18) function. This means that sub-packages will automatically be included in transpilation as well.

For more advanced scenarios users can replace the default [exclude function](https://github.com/appcelerator/appcd-plugin-webpack/blob/16cc5568bc4def2efdd627667b64620290224af5/src/config/base.js#L72) with their own implementation via a hook using [webpack-chain](https://github.com/neutrinojs/webpack-chain).